boa_ast/src/statement/labelled.rs

```diff
@@ -100,11 +101,13 @@ impl Labelled {
     }
 
     /// Gets the labelled item.
+    #[inline]
     #[must_use]
     pub const fn item(&self) -> &LabelledItem {
         &self.item
     }
 
+    #[inline]
     /// Gets the label name.
```

Comment:
```suggestion
    /// Gets the label name.
    #[inline]
```
Switched up the order by accident